xray/utils/downloadindexer.go

```diff
@@ -21,6 +26,8 @@ import (
 const (
 	indexerFileName     = "indexer-app"
 	GraphScanMinVersion = "3.29.0"
```

Comment:
Looks like this const can remain private (unless it is used by the jfrog-cli project).

Reply:
This change was merged just a few days ago (#195). It's not used by jfrog-cli yet, but it's going to be used there soon.